internal/format/base_visitor.go

```diff
@@ -152,17 +182,16 @@ func (v *baseVisitor) pInnerLiteral(name string, literal proto.Literal, suffix s
 	}
 }
 
-func (v *baseVisitor) PField(prefix string, t string, field *proto.Field) {
-	v.PComment(field.Comment)
-	if len(field.Options) == 0 {
-		v.PWithInlineComment(field.InlineComment, prefix, t, " ", field.Name, " = ", field.Sequence, ";")
+func (v *baseVisitor) PField(prefix string, fieldType string, field *proto.Field) {
+	v.pMessageOrEnumField(prefix, field.Name, fieldType, field.Sequence, field.Comment, field.InlineComment, field.Options...)
+}
+
+func (v *baseVisitor) PEnumField(element *proto.EnumField) {
```

Comment:
I'd prefer to handle the case of a `nil` `element.ValueOption` option in the `pMessageOrEnumField` func so that this can simply be:
```
v.pMessageOrEnumField("", element.Name, "", element.Integer, element.Comment, element.InlineComment, element.ValueOption)
```

But I realize that this change would also need to propagate to `pOptions`, and the change would complicate things a bit.

Eh, I'm fine with leaving this in as is.